malaria24/ona/tests/test_models.py

```diff
@@ -496,6 +496,145 @@ def test_send_digest_email(self):
         self.assertEqual(html_content.count('EHP1, EHP2'), 10)
         self.assertEqual(message.to, [mis.email_address])
 
+    @responses.activate
+    def test_print_district_digest_email_data(self):
+        Facility.objects.create(facility_code='342315',
+                                facility_name='Facility 1',
+                                province='Limpopo',
+                                district=u'Example1')
+        Facility.objects.create(facility_code='343434',
+                                facility_name='Facility 2',
+                                province='Limpopo',
+                                district=u'Example2')
+        Facility.objects.create(facility_code='333444',
+                                facility_name='Facility 3',
+                                province='Limpopo',
+                                district=u'Example3')
+        Facility.objects.create(facility_code='333333',
+                                facility_name='Facility 3',
+                                province='The Eastern Cape',
+                                district=u'Example4')
+        Facility.objects.create(facility_code='111111',
+                                facility_name='Facility 1',
+                                province='Gauteng',
+                                district=u'Example5')
+        Facility.objects.create(facility_code='333333',
+                                facility_name='Facility 3',
+                                province='The Eastern Cape',
+                                district=u'Example5')
+        manager1 = self.mk_actor(role=MANAGER_DISTRICT,
+                                 email_address='manager@example.org',
+                                 district=u'Example1', facility_code='342315')
+        self.mk_actor(role=MIS,
+                      email_address='mis@example.org',
+                      facility_code='342315', district=u'Example1')
+        self.mk_actor(role=MIS,
+                      email_address='mis@example.org',
+                      facility_code='222222', district=u'Example1')
+        ehp1 = self.mk_ehp(name='EHP1', email_address='ehp1@example.org')
+        ehp2 = self.mk_ehp(name='EHP2', email_address='ehp2@example.org')
+
+        for i in range(10):
+            case = self.mk_case(gender='female', facility_code='342315')
+            case.date_of_birth = datetime.today().strftime("%y%m%d")
+            case.ehps.add(ehp1)
+            case.ehps.add(ehp2)
+            case.save()
+            case.digest = None
+
+        for i in range(10):
+            case = self.mk_case(gender='male', facility_code='222222')
```

Comment:
I don't think you need to create these cases since you don't create a facility with this code anywhere? Unless you want to show that these cases don't get counted?